models/forecast.js

```diff
@@ -0,0 +1,12 @@
+class Forecast {
```

Comment:
@leiyakenney could you take a look at this forecast model and tell me what you think?  I made it to format the forecast data returned from darksky

Reply:
@alect47 Looks really good - I'd delete the comments out to clean it up a bit, but otherwise it looks nice and clean

Reply:
@leiyakenney Just took them out, thanks for catching that!